tests/ApiPort.Tests/AnalyzeOptionsTests.cs

```diff
@@ -1,10 +1,8 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Just to make sure, do we have tests for the other options like: listTargets, listResultFormat, docId, etc? Also, I didn't see any tests covering options like "Description", "Endpoint", etc.

it would be nice to see, but we can create an issue to add those tests.. in addition to the last set of tests from the proxy PR #531.
I think there was another PR

Reply:
We don't. I'll add some basic ones